DMS/app/src/main/java/com/example/kbb12/dms/Model/InsulinModel.java

```diff
@@ -0,0 +1,13 @@
+package com.example.kbb12.dms.model;
+
+import com.example.kbb12.dms.model.basalInsulinModel.DuplicateDoseException;
+import com.example.kbb12.dms.model.basalInsulinModel.BasalInsulinDose;
+
+import java.util.List;
+
+/**
+ * Created by kbb12 on 24/02/2017.
+ */
+public interface InsulinModel {
+    public void saveDoses(List<BasalInsulinDose> basicDoses,String basalInsulinBrandName) throws DuplicateDoseException;
```

Comment:
Does the insulin model only deal with basal insulin just now, or is this separated?